### DISCUSSION TITLE
How to cross reference supplementary tables and graphs?

### DISCUSSION THREAD

**Discussion by siganna71** [quarto-dev/quarto-cli#7755] [Q&A]
### Description

Hi, I have read that in the new pre-release 1.4 it is possible to reference supplement tables and graphs separately but I can't find instructions regarding how I can do that. Can anyone help me?
I also just want to say how happy I am to be able to cross reference in Word.
With thanks
Anna

**Comment by cscheid** [comment]
(EDIT: Removed outdated syntax)

Use the documentation: https://quarto.org/docs/authoring/cross-references-custom.html

**Reply by siganna71** [reply]
Thank you so much for this!

**Reply by siganna71** [reply]
I have a follow up question, how can I make the table caption be above the table, as in regular tables?

**Reply by cscheid** [reply]
Use `caption-location: top` on the YAML configuration for the custom crossreferenceable type.

**Reply by siganna71** [reply]
Thank you

**Comment by pedrohbraga** [comment]
@cscheid, thank you for posting this solution! I have been trying to work with it, but it has not been working:

![image](https://github.com/quarto-dev/quarto-cli/assets/8599229/6937d9c4-530e-4878-a0e9-215d5068e76e)

```
ERROR: Validation of YAML front matter failed.
ERROR: In file ClimFreqEvolSpatScales-manuscript.qmd
(line 35, columns 7--15) property name ref-type is invalid
34:       latex-env: supptbl
35:       ref-type: supptbl      
         ~~~~~~~~~
36:       caption-location: top
i The error happened in location crossref:custom:1:ref-type.
```

I removed `ref-type`, but the cross-referencing for the table does not work.

Would you have any insights on what I should do to get this to work? I will appreciate any help you can provide!

I am on Windows 11, with `"Chocolate Cosmos" Daily (58bdc18d, 2024-04-12) for windows
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) RStudio/2024.04.0-daily+693 Chrome/120.0.6099.291 Electron/28.2.6 Safari/537.36, Quarto 1.5.29`.

**Reply by cscheid** [reply]
Use this instead: https://quarto.org/docs/authoring/cross-references-custom.html You're referring to an old thread here, and the syntax has changed.

**Reply by abduazizR** [reply]
Does it work with docx? I tried it but it did not work

**Reply by cderv** [reply]
@abduazizR Open a new issue with sharing a reproducible example of what you tried. 

You can also look into existing issues to see if there is already one open for what you tried
